docs/src/main/paradox/index.md

```diff
@@ -0,0 +1,86 @@
+
+Akka HTTP follows a rather open design and many times offers several different API levels for "doing the same thing."  The users get to pick the API level of abstraction that is most suitable for their applications.  This approach means that, if you have trouble achieving something using a high-level API, there’s a good chance that you can get it done with a low-level API, which offers more flexibility but might require you to write more application code.
+
+This Hello World example demonstrates some Akka HTTP fundamentals. Within 30 minutes, you should be able to download, run the sample application, and use this documentation to guide you through this example's building blocks.
+
+## Prerequisite
+
+Having a basic understanding of Akka actors will help the reader of this guide. There is a [getting started guide](https://developer.lightbend.com/guides/akka-quickstart-scala/) for Akka actors should you feel like brushing up your knowledge thereof.
+
+## Downloading the example
+
+The Hello World example for Scala is a zipped project that includes a distribution of sbt (build tool). You can run it on Linux, MacOS, or Windows. The only prerequisite is Java 8.
+
+Download and unzip the example:
+
+1. Download the zip file from Lightbend Tech Hub by clicking `CREATE A PROJECT FOR ME`.
+2. Extract the zip file to a convenient location:
+
+* On Linux and MacOS systems, open a terminal and use the command `unzip` akka-quickstart-scala.zip. Note: On OSX, if you unzip using Archiver, you also have to make the sbt files executable:
+
+```
+$ chmod u+x ./sbt
+$ chmod u+x ./sbt-dist/bin/sbt
+```
+
+* On Windows, use a tool such as File Explorer to extract the project.
+
+## Running the example
+
+To run Hello World:
+
+In a console, change directories to the top level of the unzipped project.
+
+For example, if you used the default project name, `akka-http-quickstart-scala`, and extracted the project to your root directory, from the root directory, enter: cd akka-http-quickstart-scala
+
+Enter `./sbt` on OSX/Linux or run `sbt.bat` on Windows to start [sbt](http://www.scala-sbt.org).
+
+sbt downloads project dependencies. The `>` prompt indicates sbt has started in interactive mode.
+
+At the sbt prompt, enter `run`.
+
+sbt builds the project and runs Hello World.
+
+The output should look something like this:
+
+```
+[info] Loading global plugins from /Users/x/.sbt/0.13/plugins
+...
+[info] Running com.lightbend.akka.http.sample.QuickstartServer
+Server online at http://localhost:8080/
+Press RETURN to stop...
+```
+
+The command above will start an Akka HTTP server. We will look at how to invoke the available endpoints of the server later in this guide.
+
+Congratulations, you just ran your first Akka HTTP app. Now take a look at what happened under the covers.
```

Comment:
Maybe add a cross-link to the `Running` and `IntelliJ` chapters for people that want to jump right in and make experiments while going through the explanation.